eq-author-api/schema/resolvers/base.js

```diff
@@ -224,10 +234,17 @@ const Resolvers = {
       subscribe: () => pubsub.asyncIterator(["publishStatusUpdated"]),
     },
     commentsUpdated: {
-      resolve: async ({ pageId, questionnaire }, args, ctx) => {
+      resolve: async ({ input, questionnaire }, args, ctx) => {
         ctx.questionnaire = questionnaire;
-        const page = await getPageById(ctx, pageId);
-        return page;
+
+        const commentSubscriptionReturn = {
+          pageId: input.pageId,
+          sectionId: input.sectionId,
+          introductionId: input.introductionId,
+          confirmationId: input.confirmationId,
+        };
```

Comment:
Could rewrite this as

```
const {pageId, sectionId, introductionId, confirmationId} = input;

const commentSubscriptionReturn = {
          pageId,
          sectionId,
          introductionId,
          confirmationId,
        };
```

Feel free to ignore this, it's just a styling thing. I don't think it'll improve or decrease readability too much. It's up to you. :)